composer.json

```diff
@@ -26,20 +26,20 @@
     },
     "require": {
         "php": "~7.2.0 || ~7.3.0 || ~7.4.0",
-        "typo3/cms-core": "^9.5.4 || ^10.2",
-        "typo3/cms-extbase": "^9.5 || ^10.2",
-        "typo3/cms-fluid": "^9.5 || ^10.2",
-        "typo3/cms-frontend": "^9.5 || ^10.2"
+        "typo3/cms-core": "^9.5.4 || ^10.4",
```

Comment:
Would use "^9.5 || ^10.4" like in the other requirements. Or is there a special reason for using minimum 9.5.4?

Reply:
I don't remember. I'll test this.

Reply:
Done.